resources/js/visual-editor/editor/post-title.css

```
@@ -0,0 +1,73 @@
+    color: var(--ap-visual-editor-post-title-placeholder);
+    font-weight: 600;
+}
+
+.screen-reader-text {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    margin: -1px;
+    padding: 0;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.7.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)